src/main/java/com/es/phoneshop/model/lastViewed/LastViewed.java

```diff
@@ -0,0 +1,45 @@
+package com.es.phoneshop.model.lastViewed;
+
+import com.es.phoneshop.model.product.Product;
+
+import javax.servlet.http.HttpSession;
+import java.util.LinkedList;
+import java.util.List;
+
+public class LastViewed {
```

Comment:
Make this class a POJO, move all business logic from this class to the appropriate newly created service